packages/dumb/src/components/trends/cbg/cbg-median-animated/cbg-median-animated.tsx

```diff
@@ -36,11 +36,8 @@ import { CbgMedianTransitionMotionInterpolate } from '../../../../models/animati
 
 interface CbgMedianAnimatedProps {
   bgBounds: BgBounds
-  datum: {
-    median: number
-    msX: number
-  }
-  defaultY: number
+  median: number
+  msX: number
   showingCbgDateTraces: boolean
   sliceWidth: number
   xScale: Function
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏You could reuse the `XScale` type (I might change its name to `ScaleFunction` but it doesn't matter) defined in `x-scale.model.ts`